scripts/bytecodecompare/storebytecode.sh

```diff
@@ -82,6 +82,12 @@ EOF
 
     if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]
     then
+        if !(test -f report.txt)
+        then
+            echo "No report was created?"
+            exit 1
+        fi
+
```

Comment:
I think the block below starting with `git clone` and ending with `git push origin` should just be repeated two or three times if it fails. As far as I can tell, there are two kinds of failures:

1. race condition, some other job pushes to develop between `git clone` and `git push`
2. no changes - if the job runs twice, it would commit no changes to the repository and thus the `git commit` fails. We have to add a check for "empty commit".